utils/build.ps1

```diff
@@ -2837,6 +2848,7 @@ if (-not $SkipBuild) {
   Invoke-BuildStep Build-PackageManager $HostArch
   Invoke-BuildStep Build-Markdown $HostArch
   Invoke-BuildStep Build-Format $HostArch
+  Invoke-BuildStep Build-LMDB $HostArch
   Invoke-BuildStep Build-IndexStoreDB $HostArch
```

Comment:
You will need to modify `Build-IndexStoreDB` to pass along the path to the LMDB build.